sdk/core/core-rest-pipeline/src/policies/tracingPolicy.ts

```diff
@@ -74,19 +74,20 @@ export function tracingPolicy(options: TracingPolicyOptions = {}): PipelinePolic
 
 function tryCreateSpan(request: PipelineRequest, userAgent?: string): Span | undefined {
   try {
-    // create a new span
-    const tracingOptions: OperationTracingOptions = {
-      ...request.tracingOptions,
-      spanOptions: {
-        ...request.tracingOptions?.spanOptions,
-        kind: SpanKind.CLIENT
-      }
+    // Backwards compatible with existing APIs that carried metadata about a span in spanOptions
```

Comment:
Should we add more information related to which versions this might be relevant to?